help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,142 @@
+---
+keywords: crm;CRM;crm destinations;salesforce crm;salesforce crm destination
+title: Salesforce CRM connection
+description: The Salesforce CRM destination allows you to export your account data and activate it within Salesforce CRM for your business needs.
+---
+
+# [!DNL Salesforce CRM] connection
+
+## Overview {#overview}
+
+[Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to add Accounts and create Contacts after activating them within a new Salesforce segment for your business needs.
+
+Salesforce CRM uses [OAuth 2 with Password Grant](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/functionality/authentication/oauth2-authentication.html?lang=en) as an authentication mechanism to communicate with the Salesforce REST API.
```

Comment:
Can we rephrase to "the Salesforce CRM destination uses OAuth 2 with Password Grant as an authentication mechanism to communicate with the Salesforce REST API. Instructions are further below (link to line 83).

I suggest we remove the link you provided (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/functionality/authentication/oauth2-authentication.html?lang=en) since this link is useful for partners implementing the destination (e.g. Infosys in this case),rather than Platform users.